vision/google/cloud/vision/annotations.py

```diff
@@ -87,6 +87,9 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.annotations.Annotations`
         :returns: An instance of ``Annotations`` with detection types loaded.
         """
+        for feature_type in response.keys():
```

Comment:
You could add the filtering in the dictionary comprehension, which would avoid destroying information in the response.  E.g.:

```python
         annotations = {
             _KEY_MAP[feature_type]: _entity_from_response_type(
                 feature_type, annotation)
             for feature_type, annotation in six.iteritems(response)
            
}
```